### PR TITLE
Signed-off-by: Shehr Bano <shehr.bano@collabora.com>

### PR DESCRIPTION
**Pull request by shehr7**
Change-Id: I2d671aa591ae2283bac22097a71ed52c6ef80a47

* Resolves: # 
* Target version: master 

### Summary
As there are 3 hidden grids, changing their individual heights did not change anything so I forced change the height of ui-treeview.

### Checklist

- [x] Code is properly formatted
- [x] All commits have Change-Id
- [x] I have run tests with `make check`
- [x] I have issued `make run` and manually verified that everything looks okay
- [x] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: browser/css/jsdialogs.css [hunk 1]
```diff
@@ -14,6 +14,11 @@
 	display: flex;
 }
 
+.jsdialog.ui-treeview-body {
+	min-height: 230pt;
```
Comment (pedropintosilva) [inline browser/css/jsdialogs.css:18]: is there any specific reason for using pt here?

a good read https://www.w3.org/Style/Examples/007/units.en.html

**Review by pedropintosilva — CHANGES_REQUESTED** [review]

File: browser/css/jsdialogs.css (hunk 1 above)
Comment (shehr7) [reply browser/css/jsdialogs.css]: @pedropintosilva Thanks for this. I modified the file. Also I pulled the latest files from repo yesterday after committing, because of which the commit file shows many changes.

File: browser/css/jsdialogs.css [hunk 2]
```diff
@@ -1535,28 +1465,3 @@ div [id^='TemplateDialog'] {
...
-
-.kbd--plus {
-	color: var(--color-text-dark);
-	padding: 0 4px;
-	font-size: 0.85em;
-}
-
-.warn-copy-paste {
-	width: 100%;
-}
```
Comment (pedropintosilva) [inline browser/css/jsdialogs.css:1562]: Please fix this PR or close and create another one. There are here many unrelated code changes, it seems, as a result from solving conflicts when rebasing it locally

**Review by pedropintosilva — CHANGES_REQUESTED** [review]

File: browser/css/jsdialogs.css (hunk 2 above)
Comment (shehr7) [reply browser/css/jsdialogs.css:1562]: Noted, I am going to create a new PR as all these changes are caused after I pulled the repo
